packages/runner/src/types/tasks.ts

```diff
@@ -185,6 +185,10 @@ export interface TestOptions {
    * Tests inherit `concurrent` from `describe()` and nested `describe()` will inherit from parent's `concurrent`.
    */
   concurrent?: boolean
+  /**
+   * TODO: suite only options?
+   */
+  concurrentSuite?: boolean
```

Comment:
I don't really like having a separate option. Do we really need it?

Reply:
Oh, forgot about this TODO.
If we don't differentiate suite and task level concurrency, then single flag should suffice.
let me check it again.

Reply:
Actually I wasn't so sure about the breaking change, but this behavior probably makes sense.
Users can still make inner suite/tests sequential easily via `describe.sequential` just like described in the doc https://vitest.dev/api/#describe-sequential